deltachat-jsonrpc/src/api/types/chat.rs

```diff
@@ -239,3 +240,23 @@ impl JSONRPCChatVisibility {
         }
     }
 }
+
+#[derive(Debug, JsonSchema, TypeDef, Serialize, Deserialize)]
+pub struct EncryptionInfo {
+    /// Addresses with End-to-end encryption preferred.
+    pub mutual: Vec<String>,
+    /// Addresses with End-to-end encryption available.
+    pub no_preference: Vec<String>,
+    /// Addresses with no encryption.
+    pub reset: Vec<String>,
```

Comment:
"Reset" is just a word from the core's code, it doesn't even appear in the Autocrypt standard. I think here smth like `unavail`/`no_encryption` would be more understandable.

But maybe just use the naming for stock strings -- `encr_none`, `e2ee_available`, `e2ee_preferred`? We don't necessarily have to use the Autocrypt's wording, it's just an internal protocol, seems it's more important to be consistent with other public names.